.pre-commit-config.yaml

```
@@ -17,22 +17,14 @@ repos:
         additional_dependencies:
           - checkmate
           - digest
-          - dplyr
-          - ggplot2
-          - glue
-          - grDevices
           - lifecycle
           - logger
-          - magrittr
           - methods
-          - bioc::MultiAssayExperiment
           - R6
           - rlang
-          - rtables
           - shinyjs
```

Don't forget about the `Depends` ;)

```suggestion
          - shiny
          - shinyjs
```